dfastmi/io/RiversObject.py

```diff
@@ -42,7 +42,7 @@
 
 class RiversObject():
     branches: List[IBranch]
-    version: version # type: ignore
+    version: Version # type: ignore
```

Comment:
```suggestion
    version: Version

Reply:
visual studio code came with this remark... will update and see again

Reply:
Updated